### PR TITLE
neofs-adm: transfer funds and set roles

### LINKED ISSUE #687: Initialize side chain in neofs-adm command
- [ ] Download latest compiled version of NeoFS smart contract if `--contract` flag is not set
- [ ] Register wallets in committee
- [x] Set wallets as notary nodes
- [ ] Set wallets as alphabet nodes
- [x] Deploy NNS contract with ID:0
- [x] Deploy all NeoFS contract
- [ ] Update NNS contract
- [ ] Transfer NEO to alphabet contracts
- [ ] Transfer GAS to wallets and proxy contract

```
$ neofs-adm morph init [-c config.yaml] \
  -r https://neofs.private.corp.com:30333
  --alphabet-wallets ./alphabet-wallets
  [--contracts ./neofs-contract-v0.9.2]
  [--epoch-duration 720]
  [--max-object-size 67108864]
> Password for AZ wallet: 
> Password for BUKY wallet:
...
```

### PR DESCRIPTION
**Pull request by fyrchik**
Close #687 .

Tested with 4, 7, 15 nodes, recovering from arbitrary stage mostly works: it is possible that command will be executed and interrupted multiple times in sub-block interval. This can cost us some GAS (which goes back to committee though) but doesn't affect the outcome of the initialization.

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/initialize_transfer.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/client"
+	scContext "github.com/nspcc-dev/neo-go/pkg/smartcontract/context"
+)
+
+const (
+	// initialAlphabetNEOAmount represents total amount of NEO distributed between alphabet nodes.
+	initialAlphabetNEOAmount = native.NEOTotalSupply * 9 / 10
```
Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/initialize_transfer.go:15]: Are there any reasons to keep 1\10 of NEO supply on genesis wallet?

File: cmd/neofs-adm/internal/modules/morph/initialize_transfer.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+		transfers = append(transfers,
+			client.TransferTarget{
+				Token:   neoHash,
+				Address: to,
+				Amount:  initialAlphabetNEOAmount / int64(len(c.Wallets)),
+			},
+			client.TransferTarget{
+				Token:   gasHash,
+				Address: to,
+				Amount:  initialAlphabetGASAmount / int64(len(c.Wallets)),
```
Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/initialize_transfer.go:58]: Probably we need to decrease `initialAlphabetGASAmount`.
We suppose that admin tool will set up notary enabled environment. Therefore most of the initial GAS must go to proxy contract address later.

File: cmd/neofs-adm/internal/modules/morph/initialize_deploy.go [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/nef"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neofs-node/pkg/innerring"
+	"github.com/urfave/cli"
+)
+
+const (
+	nnsContract        = "nns"
+	neofsContract      = "neofs"
+	processingContract = "processing"
```
Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/initialize_deploy.go:24]: Processing contract deployed in main chain, ignore it.

File: go.mod [hunk 4]
```diff
@@ -11,7 +11,7 @@ require (
 	github.com/mr-tron/base58 v1.2.0
 	github.com/multiformats/go-multiaddr v0.3.2
 	github.com/nspcc-dev/hrw v1.0.9
-	github.com/nspcc-dev/neo-go v0.95.3
+	github.com/nspcc-dev/neo-go v0.96.1-pre.0.20210721163421-520133aee2f4
```
Comment (alexvanin) [inline go.mod:14]: Is it required to use unreleased version? Can we use v0.96.0?

Comment (fyrchik) [reply go.mod:14]: Actually yes, I needed this commit https://github.com/nspcc-dev/neo-go/commit/5bdcd4c241ed7e09811ce98e22e950d398220175 which seems to be in 0.96.0.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**